src/quadspline.rs

```diff
@@ -0,0 +1,20 @@
+//! Quadratic Bézier splines.
+use crate::Point;
+
+#[derive(Clone, Debug, PartialEq)]
+/// A quadratic Bézier spline.
```

Comment:
```suggestion
/// A quadratic Bézier spline.
#[derive(Clone, Debug, PartialEq)]
```

house style: docs above attributes. 🤷